spec/unit/lib/wifi_user/use_cases/sponsor_users_spec.rb

```diff
@@ -146,8 +146,11 @@ def a_plural_sponsor_confirmation_request
     it 'Sends a sponsorship failed email to the sponsor' do
       body = {
         email_address: 'cassandra@gov.uk',
-        template_id: 'efc83658-dcb5-4401-af42-e26b1945c1a9',
-        personalisation: {},
+        template_id: '52c19b68-4d8b-497a-b6ae-ee27d49439c3',
+        personalisation: {
+          # TODO: this should pick up failed sponsees in the contact sanitisor
```

Comment:
Same comment as the FIXME above.

Reply:
yeah agree with this one outright, as it won't cause confusion if seen